include/dlaf/eigensolver/backtransformation/mc.h

```diff
@@ -97,14 +96,16 @@ void BackTransformation<Backend::MC, Device::CPU, T>::call_FC(
                                                                         mat_v.distribution());
   common::RoundRobin<matrix::Panel<Coord::Row, T, Device::CPU>> panelsW2(n_workspaces,
                                                                          mat_c.distribution());
+
+  dlaf::matrix::Distribution dist_t({mb, total_reflector}, {mb, mb});
   matrix::Panel<Coord::Row, T, Device::CPU> panelT(dist_t);
 
-  const SizeType nr_reflector_blocks = dist_t.nrTiles().cols();
-  const SizeType nr_reflectors_last_block =
-      dist_t.tileSize(GlobalTileIndex(0, nr_reflector_blocks - 1)).cols();
+  const SizeType total_reflector_blocks = dist_t.nrTiles().cols();
+  const SizeType total_reflectors_last_block =

```

Comment:
I think that @albestro meant to have:
- total_nr_reflectors
- nr_reflector_blocks
- nr_reflectors
- nr_reflectors_last_block